eclipsesource/theia
# Align window.createTerminal API with VSCode

#### What it does

Deprecated PseudoTerminalOptions and created ExternalTerminalOptions to align with VSCode API.

Please let me know if this alternation from the VSCode API was done on purpose. I couldn't find a reason to not align with the VSCode API.

VSCode API for reference: https://code.visualstudio.com/api/references/vscode-api#window.

This ticket works toward resolving # 6589.

Contributed on behalf of STMicroelectronics.

#### How to test

This is rather a renaming, rather than a full change, but to ensure this still works you can use [this](https://github.com/microsoft/vscode-extension-samples/tree/main/extension-terminal-sample) sample extension.


#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
